src/pymatgen/analysis/pourbaix_diagram.py

```diff
@@ -1075,22 +1131,3 @@ def domain_vertices(self, entry):
             list of vertices
         """
         return self._pbx._stable_domain_vertices[entry]
-
-
-def generate_entry_label(entry):
```

Comment:
This should be a helper private function for `PourbaixPlotter` (as per its docstring), also as far as I'm aware, there is [no usage outside this class](PourbaixPlotter), so I might make it a private staticmethod instead